lib/cloud/aws/policy_statements.go

```diff
@@ -186,6 +186,7 @@ func StatementForEKSAccess() *Statement {
 			"eks:CreateAccessEntry",
 			"eks:DeleteAccessEntry",
 			"eks:AssociateAccessPolicy",
+			"eks:TagResource",
```

Comment:
Do we need any docs updates to account for this new permission?

Reply:
Added 👍